go-rest-example/data/platform.go

```diff
@@ -0,0 +1,33 @@
+package data
+
+import "errors"
+
+type Platform struct {
+	Name string
```

Comment:
This should have an ID to jive with my other comment above. This was just an example so I did super minimal but an ID would have helped for showing path structuring